src/pybind/mgr/dashboard/controllers/feedback.py

```diff
@@ -1,58 +1,111 @@
 # # -*- coding: utf-8 -*-
 
+from requests.exceptions import RequestException
+
+from .. import mgr
 from ..exceptions import DashboardException
-from ..model.feedback import Feedback
-from ..rest_client import RequestException
 from ..security import Scope
-from ..services import feedback
-from . import APIDoc, APIRouter, RESTController
+from . import APIDoc, APIRouter, BaseController, Endpoint, ReadPermission, RESTController, UIRouter
 
 
 @APIRouter('/feedback', Scope.CONFIG_OPT)
-@APIDoc("Feedback API", "Report")
+@APIDoc("Feedback", "Report")
 class FeedbackController(RESTController):
-    issueAPIkey = None
 
-    def __init__(self):  # pragma: no cover
-        super().__init__()
-        self.tracker_client = feedback.CephTrackerClient()
+    def list(self):
+        """
+        List all issues details.
+        """
+        try:
+            response = mgr.remote('feedback', 'get_issues')
+        except RequestException as error:
```

Comment:
AFAIK `mgr.remote` throws `RuntimeError` (or `ImportError`). Please see:
https://github.com/ceph/ceph/blob/master/src/pybind/mgr/mgr_module.py#L2106

```suggestion
        except RuntimeError as error:
```